### PR TITLE
false positive test cases for images 

### PR DESCRIPTION
**Pull request by harrisonpim**
- adds a set of false-positive test cases for images
- adds a couple of extra false-positive test cases for works
- a little bit of renaming for clarity

Screenshot 2021-05-21 at 13 37 52

### CODE REVIEW

File: data/tests/images.ts [hunk 1]
```diff
@@ -1,11 +1,13 @@
+import { equalTo0, equalTo1 } from './pass'
+
 import { Test } from '../../types'
-import { eq1 } from './pass'
+import { filterCaseRatings } from './queryAugmentation'
 
 const tests: Test[] = [
   {
-    label: 'Precision',
+    label: 'precision',
```
Comment (jamesgorrie) [inline data/tests/images.ts:8]: Any reason to lowercase there? Aesthetically I like title case, less ++programmeRx10

File: data/tests/images.ts [hunk 2]
```diff
@@ -96,6 +98,40 @@ const tests: Test[] = [
...
     },
   },
+  {
+    label: 'false-positives',
+    description:
+      "Due to fuzzy matching on alternative spellings, we need to ensure we aren't too fuzzy.",
+    pass: equalTo0,
+    searchTemplateAugmentation: filterCaseRatings,
+    cases: [
+      { query: 'monsters', ratings: ['n7r5s65w'] }, // shouldn't match "Monastery"
```
Comment (jamesgorrie) [inline data/tests/images.ts:108]: It'd be nice to have these comments showing up on the UI

**Review by jamesgorrie — APPROVED** [review]
I think I see why we've lowercases the label. 

We could be more conversational e.g. `Precision in images` or `{label} in {namespace}`. Not massively fussed, just like using labels as exactly that, which felt like they should be title case.

File: data/tests/images.ts (hunk 1 above)
Comment (harrisonpim) [reply data/tests/images.ts:8]: Yeah, your comment ☝️ is right - they're lowercased to make the combination of this label with the namespace look a bit nicer. I like the idea of eg `Precision in images` though. Will make that change now

File: data/tests/images.ts (hunk 2 above)
Comment (harrisonpim) [reply data/tests/images.ts:108]: good idea! happy for me to add a description block to each test case?

Comment (jamesgorrie) [reply data/tests/images.ts:108]: Sounds good. Probably have it per test, and then allow you to add it to the cases too.

**Review by jamesgorrie — APPROVED** [review]

**Pull request merged**